Typo in automake file.

Upstream already fixed in git & 2.33.1 release:
https://git.gnome.org/browse/gdk-pixbuf/commit/tests/Makefile.am?id=73aa6ee5d98be925365a4c3d35ecbcc61ff17ab1

--- gdk-pixbuf-2.32.1/tests/Makefile.am	Thu Oct  1 11:50:01 2015
+++ gdk-pixbuf-2.32.1/tests/Makefile.am	Thu Oct  1 11:50:15 2015
@@ -119,7 +119,7 @@
 	test-common.h			\
 	$(NULL)
 
-pixbuf_short_git_write_SOURCES =	\
+pixbuf_short_gif_write_SOURCES =	\
 	pixbuf-short-gif-write.c
 
 pixbuf_readonly_to_mutable_SOURCES =	\
